internal/config/command.go

```diff
@@ -25,10 +25,11 @@ type Command struct {
 	Exclude  string `mapstructure:"exclude"  yaml:",omitempty" json:"exclude,omitempty"  toml:"exclude,omitempty"`
 	Priority int    `mapstructure:"priority" yaml:",omitempty" json:"priority,omitempty" toml:"priority,omitempty"`
 
-	FailText    string `mapstructure:"fail_text"   yaml:"fail_text,omitempty"   json:"fail_text,omitempty"   toml:"fail_text,omitempty"`
-	Interactive bool   `mapstructure:"interactive" yaml:",omitempty"            json:"interactive,omitempty" toml:"interactive,omitempty"`
-	UseStdin    bool   `mapstructure:"use_stdin"   yaml:",omitempty"            json:"use_stdin,omitempty"   toml:"use_stdin,omitempty"`
-	StageFixed  bool   `mapstructure:"stage_fixed" yaml:"stage_fixed,omitempty" json:"stage_fixed,omitempty" toml:"stage_fixed,omitempty"`
+	FailText       string `mapstructure:"fail_text"        yaml:"fail_text,omitempty"   json:"fail_text,omitempty"        toml:"fail_text,omitempty"`
+	Interactive    bool   `mapstructure:"interactive"      yaml:",omitempty"            json:"interactive,omitempty"      toml:"interactive,omitempty"`
+	UseStdin       bool   `mapstructure:"use_stdin"        yaml:",omitempty"            json:"use_stdin,omitempty"        toml:"use_stdin,omitempty"`
+	StageFixed     bool   `mapstructure:"stage_fixed"      yaml:"stage_fixed,omitempty" json:"stage_fixed,omitempty"      toml:"stage_fixed,omitempty"`
+	FilesFromStdin bool   `mapstructure:"files_from_stdin" yaml:",omitempty"            json:"files_from_stdin,omitempty" toml:"files_from_stdin,omitempty"`
```

Comment:
I can't imagine a case when you need this option implicitly, because as I know `git` does not pass files to STDIN. So, I assume the only use case is when you explicitly run something like `echo file1 file2 | lefthook run do-something`. Do I miss something?

Reply:
With implicitly you mean, from configuration?

In the use case of Git hooks, then true. But Lefthook can be invoked in multiple ways for multiple use cases. For example, an organization may prohibit acceptance of paths via `--files` (for example, because it does have filenames with comma's in it, like data science CSV batches produced outside of their control) by enforcing this in the team-wide Lefthook config via `files_from_stdin`.

Reply:
With implicitly I mean from the git hook. If you specify `lefthook run do-something --files-from-stdin`, then you probably expect that `{files}` or `{staged_files}` are replaced from the data parsed from STDIN. But there's no need to have a special template for this case.

Reply:
Not for that case, indeed. Please note my previous reply though.

Reply:
> An organization may prohibit acceptance of paths via --files (for example, because it does have filenames with comma's in it, like data science CSV batches produced outside of their control) by enforcing this in the team-wide Lefthook config via files_from_stdin

I am sorry, I can't agree on this use case because it doesn't align with the main purpose of lefthook – git hooks management. Adding `{files_from_stdin}` template doesn't fit into my vision of what lefthook should support because it adds a very special flow not related to git hooks at all.

I am ok with this change: `--files-from-stdin` option forces lefthook to get the file list from STDIN and replace `{files}` or `{staged_files}` with the files from STDIN.

Reply:
@mrexox Can you check again whether I'm truly adding a template `{files_from_stdin}` and if so, please point out which source code sections do this, so that I can remove them while keeping a configuration option `files_from_stdin`.

Reply:
> > An organization may prohibit acceptance of paths via --files (for example, because it does have filenames with comma's in it, like data science CSV batches produced outside of their control) by enforcing this in the team-wide Lefthook config via files_from_stdin
> 
> I am sorry, I can't agree on this use case because it doesn't align with the main purpose of lefthook – git hooks management. Adding `{files_from_stdin}` template doesn't fit into my vision of what lefthook should support because it adds a very special flow not related to git hooks at all.

Then the above reflects where we stood, in our discussion.

Data science teams do record data files in Git (e.g., with Git LFS), and then these issues with commas pop up. An organizations should be able to enforce `files_from_stdin` and the only thing that can be made constant is an immutable config (file).

[This official guidelines document](https://records-express.blogs.archives.gov/2017/08/22/best-practices-for-file-naming/) warns against using commas in filenames, which means it does happen in some datasets and software should not break (indeed, a linter should be possible that detects this, this is where Lefthook would help).

Anecdotes:
- https://unix.stackexchange.com/a/155836 (search for comma)
- https://forums.autodesk.com/t5/vault-forum/filenames-with-commas/td-p/9464972